topics/MBC_cluster_conversion/mbc_divider_script.py

```diff
@@ -813,7 +815,9 @@ def main():
         host_remove_command = '/bin/sh -c "weka cluster host remove {} --no-unimprint"'.format(current_host_id)
         run_shell_command(host_remove_command)
         logger.info('the old container {} is disabled and stop, please remove it after the conversion is done'.format(container_name))
-
+        if args.remove_old_container:
+            container_rm_command = '/bin/sh -c "{}weka local rm {} -f"'.format(sudo, container_name)
```

Comment:
`{}weka` -> `{} weka`

Reply:
important!

Reply:
actually no, the sudo include the sudo string include the whitespace